doc/whatsnew.rst

```diff
@@ -46,6 +46,8 @@ v0.2.8.dev
 
 - Fix ``fit`` in :class:`pyriemann.channelselection.ElectrodeSelection`
 
+- Add example to compare classifiers on synthetic datasets
```

Comment:
You will need a what’s new entry for the bug fix in kNN too I think. Unless it was so far never released

Reply:
Done.